### ISSUE TITLE
WCAG: Viewport scaling

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2200]
Zooming in on a mobile is hampered by the viewport settings (<meta name = "viewport": user-scalable = no, max-width =, maximum-scale = 1.0 "). read content from these pages on mobile devices.

The visually impaired and visitors on mobile are missing some of the content.

Delete this code.

**Comment by maxvisser** [comment]
ask janita; browsers have option to ignore this setting; if we still need to remove it how would we need to do it?
